### ISSUE TITLE
Core BTS UI: Warning Screen

### ISSUE THREAD

**Issue by nuttycom** [zcash/secant-ios-wallet#806]
Add a warning screen to the wallet setup flow: https://www.figma.com/file/vYQD6uZtkPgtmgJ3zfauq0/Zashi-UI-082123-V3?type=design&node-id=413%3A9254&mode=design&t=MuSYkgngwcFg2nNl-1

**Comment by LukasKorba** [comment]
<img width="334" alt="Screenshot 2023-08-22 at 7 47 33" src="https://github.com/zcash/secant-ios-wallet/assets/7198042/ec83d58b-cc60-49ae-aa2f-404c9994c969">

This is a brand new screen so it must be built from scratch. 

Points to focus on:
- Please make sure this is done as a new independent module in Features folder
- put the texts into the Features/Generated/Localizable.strings and reference them with SwiftGen L10n
- this screen must precede generation of the seed and all other logic tight to it, only after users go from this screen to the next one (recovery phrase screen) it should do the logic. Tapping the button `create new wallet` on `PlainOnboardingView` triggers RootInitialization.swift's `case .initialization(.createNewWallet):` but this should be postponed by one step

@nuttycom to confirm:
- the hamburger menu should probably be back button at this screen, allowing users to go back and potentially change the decision from create new wallet to restore it from the recovery phrase. Hamburger menu puts the feeling that more options is available but I believe it's not
- I see no button how users go from this screen to the next one, the only UI element is the toggle `I acknowledge`. If tapping this toggle means `confirm and go to the next screen` it's not probably the best way, usually users can turn on and off toggles again before they confirm via some other button. I see 2 options how to solve this, either unhide a button when the toggle is turned on (hide when off again) OR the button is always present but disabled till the toggle is on.

**Comment by LukasKorba** [comment]
Resolved in #843
